src/features/bounds/bounds-manipulation.ts

```diff
@@ -39,10 +40,16 @@ export class SetBoundsAction implements Action {
  * This hidden rendering round-trip is necessary if the client is responsible for parts of the layout
  * (see `needsClientLayout` viewer option).
  */
-export class RequestBoundsAction implements Action {
-    readonly kind = RequestBoundsCommand.KIND;
+export class RequestBoundsAction implements RequestAction<ComputedBoundsAction> {
+    static readonly KIND: string  = 'requestBounds';
+    readonly kind = RequestBoundsAction.KIND;
 
-    constructor(public readonly newRoot: SModelRootSchema) {
+    constructor(public readonly newRoot: SModelRootSchema,
+                public readonly requestId: string = '') {}
+
+    /** Factory function to dispatch a request with the `IActionDispatcher` */
+    static create(newRoot: SModelRootSchema): RequestAction<ComputedBoundsAction> {
+        return new RequestBoundsAction(newRoot, generateRequestId());
```

Comment:
Currently we mark all properties of actions as `readonly`. Adding the request id after action creation would break that contract.

We need such a static function anyway for each request action because the TypeScript type system is currently unable to match the correct type parameter for a class implementing a parameterized interface (it does work in Java).